Userland/Services/Taskbar/TaskbarWindow.cpp

```diff
@@ -101,7 +101,7 @@ TaskbarWindow::TaskbarWindow(NonnullRefPtr<GUI::Menu> start_menu)
 
 void TaskbarWindow::config_string_did_change(String const& domain, String const& group, String const& key, String const& value)
 {
-    VERIFY(domain == "Taskbar");
+    VERIFY(domain == "Taskbar" || domain == "Calendar");
     if (group == "Clock" && key == "TimeFormat") {
```

Comment:
The `if` statement here should also be checking for `domain == "Taskbar"`, because otherwise it will react to either `Taskbar::Clock::TimeFormat` or `Calendar::Clock::TimeFormat`.